addon/mixins/route.js

```diff
@@ -193,6 +199,7 @@ const RouteMixin = Ember.Mixin.create({
       currentPage: startingPage,
       _firstPageLoaded: false,
       _perPage: perPage,
+      _store: store,
```

Comment:
@Redsandro We also probably need to be able to customize the `_storeFindMethod` method.  What are your thoughts on this?  Does your custom store use the same `query` method?